### PR TITLE
Add weather sounds

### LINKED ISSUE #29: Add weather sounds
Adding appropriate sounds can make the user experience more immersive.
- If the weather is windy, play wind sounds.
- If the weather is rainy, play rain sounds, etc

Add a button for the user to toggle between sound on and off

### PR DESCRIPTION
**Pull request by adizcode**
Plays sound for snow, rain, thunderstorm, and drizzle weather conditions upon clicking the floating button

Closes #29 

Authors:
@anubhav06 and @adizcode

### CODE REVIEW

File: example.env [hunk 1]
```diff
@@ -1 +0,0 @@
-REACT_APP_APIKEY=
```
Comment (ditsuke) [inline example.env:1]: We need to restore this

File: src/Sound.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import { Howl } from 'howler';
```
Comment (ditsuke) [inline src/Sound.js:1]: Should be moved to `src/components`

File: src/index.css [hunk 3]
```diff
@@ -11,3 +11,69 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
```
Comment (ditsuke) [inline src/index.css:14]: These CSS changes should be moved to `src/Components/WeatherSounds/WeatherSounds.css` and imported into the component.

**Review by ditsuke — CHANGES_REQUESTED** [review]

File: example.env (hunk 1 above)
Comment (adizcode) [reply example.env:1]: Okay, I'll make the changes. Are these the things that are causing tests to fail?

Comment (ditsuke) [reply example.env:1]: No, that's the code style (prettier). We'll need a `yarn format` run to fix that

Comment (adizcode) [reply example.env:1]: Can you take a look now? :D

**Review by yyjlincoln — APPROVED** [review]
Hey - this is looking great. There's one issue though - it doesn't work on the deployment preview:

> The AudioContext was not allowed to start. It must be resumed (or created) after a user gesture on the page. https://goo.gl/7K7WLu

I think it's a Chrome policy or something. Does it work on localhost? Do we need to change anything?

**Comment by anubhav06** [comment]
It works for me as well. @yyjlincoln maybe you didn't switch the toggle to ON ?

**Comment by ditsuke** [comment]
The button seems to reset every time a the search location is changed, is that expected behavior?

**Comment by adizcode** [comment]
> The button seems to reset every time a the search location is changed, is that expected behavior?

That's a valid point, but also the component is rebuilt on every new location. So, it's expected.

**Comment by gmelodie** [comment]
Don't hear anything on Firefox (searched then turned the button on). Also the button feels very cryptic (hard to understand what it does).

**Comment by yyjlincoln** [comment]
> Don't hear anything on Firefox (searched then turned the button on). Also the button feels very cryptic (hard to understand what it does).

Had the same issue - I checked the console and it was the error above

**Pull request merged**